### ISSUE TITLE
[component] Title : Support using Altair/RunTime Network Computer tool for scheduling build / run jobs in parallel in dvsim

### ISSUE THREAD

**Issue by venkatk-ot** [lowRISC/opentitan#24628]
### Description

Give a description of the problem, listing similar issues and if able:
- Assign the issue to someone familiar with it.
- Add appropriate labels.

**Comment by venkatk-ot** [comment]
Currently local, sge and lsf schedulers are supported. Altair / RunTime NC is another popular tool

**Comment by venkatk-ot** [comment]
new pull request available here
https://github.com/lowRISC/opentitan/pull/24635
